templates/question.html

```diff
@@ -527,14 +527,17 @@ <h1>{{title|safe}}</h1>
                     {% endif %}
                     &raquo;
                   </button>
-
+                </div>
+                <div>
+                  {% if back_url %}
+                    <a href="{{back_url}}" id="back-button" class="btn btn-link question-button">previous question</button>
+                  {% endif %}
```

Comment:
```suggestion
                  {% if back_url %}
                      <div class="pull-left">
                          <button id="back-button" class="btn  btn-default btn-link question-button"><a href="{{back_url}}" style="color: inherit;">previous question</a></button>
                      </div>
                  {% endif %}
```

Reply:
@davidpofo I partially accepted suggestion. Moved back button to left but modified styling different than suggested.